### PR TITLE
Update outdated documentation regarding new experiment version changes

### PR DESCRIPTION
**Pull request by DatCorno**

### CODE REVIEW

File: docs/src/user/cli/info.rst [hunk 1]
```diff
@@ -6,10 +6,16 @@ Here is an example of all the sections provided by the command.
 
 .. code-block:: console
 
-   orion info orion-tutorial
+   orion info --name orion-tutorial
```
Comment (bouthilx) [inline docs/src/user/cli/info.rst]: Unless this was changed, the command is `orion info <exp name>` directly.

File: docs/src/user/cli/info.rst [hunk 2]
```diff
@@ -59,3 +65,31 @@ Here is an example of all the sections provided by the command.
 
 The last section contains information about the best trial so far, providing its
 hyperparameter values and the corresponding objective.
+
+The ``--name`` argument
+~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (bouthilx) [inline docs/src/user/cli/info.rst]: Unless the command has been changed this should be removed.

File: docs/src/user/cli/info.rst [hunk 3]
```diff
@@ -59,3 +65,31 @@ Here is an example of all the sections provided by the command.
...
+To get information on an experiment, you need to call `info` with the `--name` or `-n` argument like
+shown in the previous example. This will fetch the latest version of the experiment with that name
+inside the database and display its content.
+
+The ``--version`` argument
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+To specify which version of an experiment you wish to observe, you can use the `--version` argument.
+If provided, this will fetch the experiment with a version number corresponding to that version
+instead of fetching the latest one. Note that the `--version` argument cannot be used alone and that
```
Comment (bouthilx) [inline docs/src/user/cli/info.rst]: For info it can be used alone because there is no `--name` argument.

File: docs/src/user/cli/info.rst [hunk 4]
```diff
@@ -59,3 +65,31 @@ Here is an example of all the sections provided by the command.
...
+an invalid version number, i.e. a version number greater than the latest version, will simply fetch
+the latest one instead.
+
+For example, suppose we have two experiments named `orion-tutorial` inside the database, one with
+version `1` and one with version `2`. Then running the following command would simply give us the
+latest version, so version `2`.
+
+.. code-block:: console
+
+   orion info --name orion-tutorial
```
Comment (bouthilx) [inline docs/src/user/cli/info.rst]: Should remove `--name`

File: docs/src/user/cli/info.rst [hunk 5]
```diff
@@ -59,3 +65,31 @@ Here is an example of all the sections provided by the command.
...
+
+Whereas, running the next command will give us the first version instead:
+
+.. code-block:: console
+
+   orion info --name orion-tutorial --version 1
```
Comment (bouthilx) [inline docs/src/user/cli/info.rst]:
```suggestion
   orion info orion-tutorial --version 1
```

File: docs/src/user/cli/info.rst (hunk 1 above)
Comment (DatCorno) [reply docs/src/user/cli/info.rst]: It was changed.

File: docs/src/user/cli/info.rst (hunk 2 above)
Comment (DatCorno) [reply docs/src/user/cli/info.rst]: It was changed.

File: docs/src/user/cli/info.rst (hunk 3 above)
Comment (DatCorno) [reply docs/src/user/cli/info.rst]: There is.

**Comment by bouthilx** [comment]
After offline discussion we decided we would make the first positional argument to be the experiment's name for all relevant commands. This will be an optional positional argument.

**Comment by bouthilx** [comment]
[Update doc for --version](https://trello.com/c/GBeAAPjC/271-update-doc-for-version)

**Pull request merged**